app/controllers/mixins/ems_common_angular.rb

```diff
@@ -431,6 +431,7 @@ def build_credentials(ems, mode)
          ems.supports_authentication?(:bearer) && !params[:default_password].blank?
         creds[:hawkular] = {:auth_key => params[:default_password], :userid => "_"}
         creds[:bearer] = {:auth_key => params[:default_password]}
+        creds.delete(:default)
```

Comment:
@Fryguy 
> we should not be creating a default auth at all if it's not used

the `default` authentication is created if `params[:default_userid]` exists:
https://github.com/ManageIQ/manageiq/pull/9947/files#diff-f402e411cd6dc158c724b14d72ead612L399
I can add an `&& !ems.kind_of?(ManageIQ::Providers::ContainerManager)` in line 399, but it will not make this more pretty :cry: